Compilation error

Hello, I can't seem to compile this project.
I've manually compiled (`make`) and installed `sudo make install`) `argparser`, `auto-auto-complete` and `libkeccak`. I believe I have all of the required dependencies.
I am running Ubuntu 16.04.

```
$ make
cc -Wall -Wextra -pedantic -Wdouble-promotion -Wformat=2 -Winit-self -Wmissing-include-dirs -Wtrampolines -Wfloat-equal -Wshadow -Wmissing-prototypes -Wmissing-declarations -Wredundant-decls -Wnested-externs -Winline -Wno-variadic-macros -Wswitch-default -Wpadded -Wsync-nand -Wunsafe-loop-optimizations -Wcast-align -Wstrict-overflow -Wdeclaration-after-statement -Wundef -Wbad-function-cast -Wcast-qual -Wlogical-op -Wstrict-prototypes -Wold-style-definition -Wpacked -Wvector-operation-performance -Wunsuffixed-float-constants -Wsuggest-attribute=const -Wsuggest-attribute=noreturn -Wsuggest-attribute=pure -Wsuggest-attribute=format -Wnormalized=nfkc -std=gnu99 -O3 -c -o obj/keccak-224sum.o src/keccak-224sum.c
cc -Wall -Wextra -pedantic -Wdouble-promotion -Wformat=2 -Winit-self -Wmissing-include-dirs -Wtrampolines -Wfloat-equal -Wshadow -Wmissing-prototypes -Wmissing-declarations -Wredundant-decls -Wnested-externs -Winline -Wno-variadic-macros -Wswitch-default -Wpadded -Wsync-nand -Wunsafe-loop-optimizations -Wcast-align -Wstrict-overflow -Wdeclaration-after-statement -Wundef -Wbad-function-cast -Wcast-qual -Wlogical-op -Wstrict-prototypes -Wold-style-definition -Wpacked -Wvector-operation-performance -Wunsuffixed-float-constants -Wsuggest-attribute=const -Wsuggest-attribute=noreturn -Wsuggest-attribute=pure -Wsuggest-attribute=format -Wnormalized=nfkc -std=gnu99 -O3 -c -o obj/common.o src/common.c
cc -Wall -Wextra -pedantic -Wdouble-promotion -Wformat=2 -Winit-self -Wmissing-include-dirs -Wtrampolines -Wfloat-equal -Wshadow -Wmissing-prototypes -Wmissing-declarations -Wredundant-decls -Wnested-externs -Winline -Wno-variadic-macros -Wswitch-default -Wpadded -Wsync-nand -Wunsafe-loop-optimizations -Wcast-align -Wstrict-overflow -Wdeclaration-after-statement -Wundef -Wbad-function-cast -Wcast-qual -Wlogical-op -Wstrict-prototypes -Wold-style-definition -Wpacked -Wvector-operation-performance -Wunsuffixed-float-constants -Wsuggest-attribute=const -Wsuggest-attribute=noreturn -Wsuggest-attribute=pure -Wsuggest-attribute=format -Wnormalized=nfkc -std=gnu99  -lkeccak -largparser -o bin/keccak-224sum obj/keccak-224sum.o obj/common.o
obj/keccak-224sum.o: In function `main':
keccak-224sum.c:(.text.startup+0x60): undefined reference to `args_standard_abbreviations'
keccak-224sum.c:(.text.startup+0x65): undefined reference to `args_init'
obj/common.o: In function `generalised_sum_fd_hex':
common.c:(.text+0x45): undefined reference to `libkeccak_state_initialise'
common.c:(.text+0x128): undefined reference to `libkeccak_fast_update'
common.c:(.text+0x196): undefined reference to `libkeccak_fast_digest'
obj/common.o: In function `hash':
common.c:(.text+0x23e): undefined reference to `libkeccak_generalised_sum_fd'
common.c:(.text+0x308): undefined reference to `libkeccak_fast_squeeze'
common.c:(.text+0x317): undefined reference to `libkeccak_squeeze'
obj/common.o: In function `check_checksums':
common.c:(.text+0x7a4): undefined reference to `libkeccak_unhex'
obj/common.o: In function `print_checksum':
common.c:(.text+0x945): undefined reference to `libkeccak_behex_lower'
common.c:(.text+0x985): undefined reference to `libkeccak_behex_upper'
obj/common.o: In function `run':
common.c:(.text+0x9e3): undefined reference to `args_new_argumentless'
common.c:(.text+0xa15): undefined reference to `args_add_option'
common.c:(.text+0xa44): undefined reference to `args_new_argumented'
common.c:(.text+0xa8e): undefined reference to `args_add_option'
common.c:(.text+0xab8): undefined reference to `args_new_argumented'
common.c:(.text+0xb02): undefined reference to `args_add_option'
common.c:(.text+0xb36): undefined reference to `args_new_argumented'
common.c:(.text+0xb83): undefined reference to `args_add_option'
common.c:(.text+0xbb7): undefined reference to `args_new_argumented'
common.c:(.text+0xc04): undefined reference to `args_add_option'
common.c:(.text+0xc33): undefined reference to `args_new_argumented'
common.c:(.text+0xc7d): undefined reference to `args_add_option'
common.c:(.text+0xca7): undefined reference to `args_new_argumented'
common.c:(.text+0xcf1): undefined reference to `args_add_option'
common.c:(.text+0xd20): undefined reference to `args_new_argumentless'
common.c:(.text+0xd6b): undefined reference to `args_add_option'
common.c:(.text+0xd9a): undefined reference to `args_new_argumentless'
common.c:(.text+0xde5): undefined reference to `args_add_option'
common.c:(.text+0xe0a): undefined reference to `args_new_argumentless'
common.c:(.text+0xe57): undefined reference to `args_add_option'
common.c:(.text+0xe81): undefined reference to `args_new_argumentless'
common.c:(.text+0xecc): undefined reference to `args_add_option'
common.c:(.text+0xef1): undefined reference to `args_new_argumentless'
common.c:(.text+0xf3e): undefined reference to `args_add_option'
common.c:(.text+0xf63): undefined reference to `args_new_argumentless'
common.c:(.text+0xfb0): undefined reference to `args_add_option'
common.c:(.text+0xfbe): undefined reference to `args_parse'
common.c:(.text+0xfc8): undefined reference to `args_opts_used'
common.c:(.text+0xfda): undefined reference to `args_opts_used'
common.c:(.text+0xfec): undefined reference to `args_opts_used'
common.c:(.text+0xffe): undefined reference to `args_opts_used'
common.c:(.text+0x1010): undefined reference to `args_opts_used'
obj/common.o:common.c:(.text+0x1022): more undefined references to `args_opts_used' follow
obj/common.o: In function `run':
common.c:(.text+0x10ab): undefined reference to `libkeccak_degeneralise_spec'
common.c:(.text+0x10d4): undefined reference to `args_opts_used'
common.c:(.text+0x1135): undefined reference to `args_dispose'
common.c:(.text+0x119e): undefined reference to `args_opts_get'
common.c:(.text+0x11ab): undefined reference to `args_opts_get_count'
common.c:(.text+0x11d6): undefined reference to `args_opts_get'
common.c:(.text+0x11e3): undefined reference to `args_opts_get_count'
common.c:(.text+0x120e): undefined reference to `args_opts_get'
common.c:(.text+0x121b): undefined reference to `args_opts_get_count'
common.c:(.text+0x1246): undefined reference to `args_opts_get'
common.c:(.text+0x1253): undefined reference to `args_opts_get_count'
common.c:(.text+0x127e): undefined reference to `args_opts_get'
common.c:(.text+0x128b): undefined reference to `args_opts_get_count'
common.c:(.text+0x12b6): undefined reference to `args_opts_get'
common.c:(.text+0x12c3): undefined reference to `args_opts_get_count'
common.c:(.text+0x12e5): undefined reference to `args_help'
common.c:(.text+0x12ea): undefined reference to `args_dispose'
common.c:(.text+0x1495): undefined reference to `args_dispose'
common.c:(.text+0x1504): undefined reference to `args_dispose'
collect2: error: ld returned 1 exit status
Makefile:83: recipe for target 'bin/keccak-224sum' failed
make: *** [bin/keccak-224sum] Error 1
rm obj/keccak-224sum.o obj/common.o
```

--- a/Makefile
+++ b/Makefile
@@ -81,7 +81,7 @@ command: $(foreach C,$(CMDS),bin/$(C))
 
 bin/%: obj/%.o obj/common.o
 	@mkdir -p bin
-	$(CC) $(FLAGS) $(LDOPTIMISE) -lkeccak -largparser -o $@ $^ $(LDFLAGS)
+	$(CC) $(FLAGS) $(LDOPTIMISE) -o $@ $^ $(LDFLAGS) -lkeccak -largparser
 
 obj/%.o: src/%.c src/*.h
 	@mkdir -p obj

